src/components/radio-button/radio-button-group.style.ts

```diff
@@ -1,5 +1,27 @@
 import styled, { css } from "styled-components";
 
+export const ErrorBorder = styled.span`
+  ${({ warning }: { warning: boolean }) =>
+    css`
+      position: absolute;
+      z-index: 6;
+      width: 2px;
+      background-color: ${warning
+        ? "var(--colorsSemanticCaution500)"
+        : "var(--colorsSemanticNegative500)"};
+      left: -12px;
+      bottom: -2px;
```

Comment:
Question: Do we need to bottom and top to not be 0px here? I ask as we already have these defined in `textbox.style` maybe they can be used here or at least extended and the necessary styles overridden.

```
const CheckableErrorBorder = styled(ErrorBorder)`
bottom: -2px;
top: 2px;
`
```

Reply:
In the designs it looks like the error border extends slightly below the last radio button. Having bottom as 0px makes it look like the border stops too short. I'm happy to run it past UX though to see if 0px would suffice. Using the `ErrorBorder` from textbox might be a good idea even if we just need to potentially override the bottom css value.

Reply:
No worries, yeah I couldn't decide if that was the case based on the designs myself so maybe bounce it off UX and go from there